joomla/joomla-cms
# Avoid conflict between JQTooltip in the same DOM as a BSTooltip

Pull Request for Issue Closes #15896.

### Summary of Changes

Allow JQTooltip in the same DOM as a BSTooltip without JS errors when destorying tooltips in mediafield javascript

### Testing Instructions

To setup:
Add this jQueryUI code to your /administrator/templates/isis/index.php before </head>
```html
<script src="https://code.jquery.com/ui/1.12.0/jquery-ui.min.js" integrity="sha256eGE6blurk5sHj+rmkfsGYeKyZx3M4bG+ZlFyA7Kns7E=" crossorigin="anonymous"></script>
```

Replicate the error in the JS console

![screen shot 2017-05-10 at 20 50 07](https://cloud.githubusercontent.com/assets/400092/25918163/5eb7d664-35c2-11e7-83ae-44ecaba1f1cb.png)

To Test: 
Apply PR (leaving jQueryUI in isis/index.php) 
See Error disappears 
Check this works with debug OFF and debug ON in Joomla Config. (one uses minified JS and one doesnt)

### Expected result

No console errors


### Documentation Changes Required

None
